ui/src/app/settings/components/repos-list/repos-list.tsx

```diff
@@ -284,84 +294,165 @@ export class ReposList extends React.Component<
                 }}>
                 <div className='repos-list'>
                     <div className='argo-container'>
-                        <DataLoader load={() => services.repos.list()} ref={loader => (this.repoLoader = loader)}>
-                            {(repos: models.Repository[]) =>
-                                (repos.length > 0 && (
-                                    <div className='argo-table-list'>
-                                        <div className='argo-table-list__head'>
-                                            <div className='row'>
-                                                <div className='columns small-1' />
-                                                <div className='columns small-1'>TYPE</div>
-                                                <div className='columns small-2'>NAME</div>
-                                                <div className='columns small-2'>PROJECT</div>
-                                                <div className='columns small-4'>REPOSITORY</div>
-                                                <div className='columns small-2'>CONNECTION STATUS</div>
-                                            </div>
-                                        </div>
-                                        {repos.map(repo => (
-                                            <div
-                                                className={`argo-table-list__row ${this.isRepoUpdatable(repo) ? 'item-clickable' : ''}`}
-                                                key={repo.repo}
-                                                onClick={() => (this.isRepoUpdatable(repo) ? this.displayEditSliding(repo) : null)}>
+                        <div style={{display: 'flex', margin: '20px 0', justifyContent: 'space-between'}}>
+                            <div style={{display: 'flex', gap: '8px'}}>
```

Comment:
```suggestion
                            <div style={{display: 'flex', gap: '8px', width:  '50%'}}>
```
this is for the dropdown anchor spacing